src/routes/index.ts

```diff
@@ -7,15 +7,25 @@ import {
   updateUser,
 } from '../controllers/user.controller';
 import { validateSchema, UserSchema } from '../validations/user.validation'
+import { LogoutUsers } from '../controllers/index';
+
+
+
+
 
 const router = express.Router();
 
 export default (): express.Router => {
   router.post('/create', validateSchema(UserSchema.signUp), registerUser);
+  router.get("/logout", LogoutUsers);
   router.get('/users', getAllUsers);
   router.get('/:id', getUserById);
   router.patch('/:id', updateUser);
   router.delete('/:id', deleteUser);
 
+
   return router;
-};
\ No newline at end of file
+};
```

Comment:
Please rebase, none of this is supposed to be here. Each feature has its own route file and this is for exporting the routers in the route files. We should have auth, user, etc.. route files